src/frontends/paddle/src/op/linspace.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/frontend/paddle/visibility.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs linspace(const NodeContext& node) {
+    auto start = node.get_input("Start");
```

Comment:
How about the inputs are variables, but not a tensor. e.g int or float
![image](https://user-images.githubusercontent.com/91237924/223651579-69ca44e3-a466-41d4-9dce-5620939d07fd.png)